be2-scala/src/main/scala/ch/epfl/pop/model/objects/RollCallData.scala

```diff
@@ -0,0 +1,40 @@
+package ch.epfl.pop.model.objects
+
+import ch.epfl.pop.json.MessageDataProtocol._
+import ch.epfl.pop.model.network.Parsable
+import ch.epfl.pop.model.network.method.message.Message
+import ch.epfl.pop.model.network.method.message.data.ActionType.ActionType
+import ch.epfl.pop.model.network.method.message.data.rollCall.{CloseRollCall, CreateRollCall, OpenRollCall, ReopenRollCall}
+import spray.json._
+
+final case class RollCallData(update_id: Hash, state: ActionType) {
```

Comment:
can you rename this field to `update_id`? This is a code smell

Reply:
I do not understand: isn’t it already already `update_id` ?